app-admin/etcd-wrapper/files/etcd-wrapper

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e
+
+USAGE="
+Usage: $0 [etcd flags]
+
+Wrapper for launching etcd via rkt.
+
+The version of etcd to be used can be set through ETCD_VERSION. The version
+must be set to one of the image tags published here:
+
+  https://quay.io/repository/coreos/etcd?tab=tags
+
+Alternatively, the keywords 'alpha', 'beta', and 'stable' may be used to specify
+the version. If no version is provided, the CoreOS update group is used.
+"
+
+case ${1} in
+	-h | --help)
+		echo "${USAGE}"
+		exit 0
+		;;
+esac
+
+source /usr/share/coreos/update.conf
+source /etc/coreos/update.conf
+
+ETCD_VERSION="${ETCD_VERSION:-${GROUP:-stable}}"
+ETCD_ACI="${ETCD_ACI:-coreos.com/etcd}"
+
+case ${ETCD_VERSION} in
```

Comment:
If etcd updates are being tied to coreos updates kinda/sorta as are done here to avoid updating etcd by surprise or mixed cluster versions, piggy backing on locksmith/update_engine for coordination, then instead we need to resurrect the rkt container updater project and implement per-container coordination. Either components are in containers and maintained independently of the OS, or the component is simply included in the OS image. kinda sorta doing both means we get to combine the problems of both into one unfortunate pile of sadness.